scripts/release/02-bump-versions

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+if [[ "$#" -ne 1 ]]; then
+  echo "$0: VERSION"
+  exit 1
+fi
+
+SCRIPTDIR=`dirname "$0"`
+VERSION="$1"
+pushd $SCRIPTDIR
+
+python ./bump_versions.py --oso_version "${VERSION}" \
```

Comment:
maybe fine to expect folks to have `packaging` & `tomlkit` installed (idk what's normal in Pythonland) but I had to install both when running this. for ✧･ﾟ: *✧･ﾟ:* extreme hands-off-ed-ness *:･ﾟ✧*:･ﾟ✧ we could pip install them here

Reply:
yeah we could, it's required

Reply:
I think installing deps once is better than doing it every time

Reply:
also this is mega hack